### ISSUE TITLE
Pill Ideas

### ISSUE THREAD

**Issue by Sebeb** [dhubber/mazed-up#15]
We're doing fine for pills in terms of quantity. But if you have any amazing ideas for pills which could really add a new or interesting twist on gameplay, let's gather them all here.

**Comment by dhubber** [comment]
I've kind of lost track of what all the pills are, what they do, etc.. .  Could we list them all here as a discussion place for helping to think of new pill ideas and possibly removing any others?  Maybe I should go through it myself actually since you know them better than me already.

**Comment by dhubber** [comment]
1. Awkward Pill
2. Bad Trip Pill
3. Blurry Pill
4. Darkness Pill
5. Death Pill
6. First Person Pill
7. Ghostly Enemy Pill
8. Ghostly Player Pill
9. Hot Walls Pill
10. Ice Pill
11. Isometric Pill
12. JPEG Pill
13. Regeneration Pill
14. Reverse Pill
15. Rotation Pill
16. Slow Pill
17. Speed Pill
18. Tank Pill
19. Zoom Pill

Think that's correct!

**Comment by Sebeb** [comment]
Yep, that's all of them. Btw, have you discovered the little device menu I made in the editor for quickly spawning in pills? It's very useful for quickly testing side effects.
As it stands, are there any pills which you think just make the game worse and shouldn't be in it?

**Comment by dhubber** [comment]
No, I didn't discover it yet but will look for it next time I want to play with specific pills for any reason. I wouldn't say I feel any of the pills make the game worse to the extent that they need to be removed.  But some are not executed in the best way and need some modifications.  For example, here are a few suggested improvements to those listed there : 
3. Blurry Pill : Perhaps a bigger range of blurring and maybe it can change on a slightly slower timescale (but only slightly).
7. Ghostly Enemy Pill : Maybe I'm remembering wrong but do the ghosts not fade completely anymore?  If they don't, I'd prefer that they did.
12. JPEG Pill : Similar to Blurry Pill, maybe it needs to be a tad stronger
15. Rotation Pill : I liked the original where the controls still pointed in the same direction relative to the maze, since it made it slowly harder and eventually the player could even forget the orientation. So I'm not sure yet which one is best but one to consider.  Also, there seems to be some problem with this in combination with other pills (e.g. first person pill)
18. Tank Pill : As you put in another issue, there's definitely some problem with this one.

**Comment by dhubber** [comment]
Btw, I remember you asking about a 20th pill.  One possibility goes back to the original jam where we wanted other visual pills, like a 'fog of war' pill or one that made distortion, like television interference or something like that.
